tests/test_lstchain.py

```diff
@@ -1,5 +1,7 @@
 from ctapipe.utils import get_dataset_path
 import numpy as np
+from astropy.coordinates import EarthLocation
+EarthLocation._get_site_registry(force_download=True)
```

Comment:
why do you want to add the EarthLocation to the lstchain test? This seems like a general functionality that could be better suited, if necessary, to ctapipe

Reply:
There seems to be an issue with astropy recently where the registry is not _always_ downloaded.
I have had a lot of trouble tracking the issue because it does not seem to be fully reproducible (or at least I could not).
See the following [Travis build](https://travis-ci.org/cta-observatory/cta-lstchain/builds/525712712) in a recent PR that failed because
```
astropy.coordinates.errors.UnknownSiteException: "Site 'Roque de los Muchachos' not in database. Use the 'names' attribute to see available sites."
```

I needed to force the download to fix it. I am investigating this but in the meantime added it to have a fix.

Reply:
You should probably just define a constant `ORM` with fixed values instead of relying on astropy's database.